tools/src/format.rs

```diff
@@ -140,15 +131,27 @@ pub fn format_code(jumps: &HashMap<String, Jump>, format: FormatAs,
 
                 let menu_jumps = menu_jumps.into_iter().map(|(_, v)| v).collect::<Vec<_>>();
 
+                // Build the list of symbols for the highlighter.
                 let index = generated_json.len();
+                let syms = {
+                    let mut syms = String::new();
+                    for (i, sym) in d.iter().flat_map(|item| item.sym.iter()).enumerate() {
```

Comment:
Can't you do `d.iter().flat_map(|item| item.sym.iter()).join(",")` ?

Reply:
Not without cloning all the symbols, but maybe that's ok.